lib/utils/analyticsConstants.js

```diff
@@ -5,6 +5,10 @@ export const REFERRAL_CLICK_WEBSITE = 'Click Referral Website';
 export const SERVICE_CLICK_WEBSITE = 'Click Service Website';
 export const REFERRAL_SUBMIT_SUCCESS = 'Submit Referral (Success)';
 export const REFERRAL_SUBMIT_INVALID = 'Submit Referral (Invalid)';
+export const VIEW_SUMMARY_EMAIL_CLICKED = 'Click View Summary Email';
+export const SEND_SUMMARY_SUCCESS = 'Send Summary (Success)';
+export const SEND_SUMMARY_INVALID_COUNT = 'Summary Services Count (Invalid)';
+export const SEND_SUMMARY_SUCCESS_COUNT = 'Summary Services Count';
```

Comment:
We should probably set line 11 to 'Summary Service count (Success)' to match the others. It may not look clear it's a success in reports alongside the others.

Reply:
Ahh yes good point!